benchmarks/Cases/BenchmarkZstdIgbinary.php

```diff
@@ -57,27 +59,32 @@ public function seed(): void
      */
     protected function setSerialization($client): void
     {
-        $client->setOption(Redis::OPT_SERIALIZER, Redis::SERIALIZER_IGBINARY);
-        $client->setOption(Redis::OPT_COMPRESSION, Redis::COMPRESSION_ZSTD);
+        $client->setOption(Relay::OPT_SERIALIZER, Relay::SERIALIZER_IGBINARY);
+        $client->setOption(Relay::OPT_COMPRESSION, Relay::COMPRESSION_ZSTD);
```

Comment:
Can you test this with PhpRedis that was NOT compiled with ZSTD or IGBINARY support to see what happens?

Reply:
In this case PhpRedis just returns `false`  for `setOption` call

Reply:
done